helm/templates/config-bootstrapper-config.yaml

```diff
@@ -1,45 +1,51 @@
+{{ $global := . }}
```

Comment:
What does this do and why do we need it? Just curious

Reply:
This is just temporary variable which is encapsulating the context. 
Inside this block `{{- range $k, $job := .Values.jobs }}` the scope is limited and you can't access variables using `.Values...`, so everything outside scope of the loop has to be accessed using this variable.

Reply:
you can just use `$.Values....`
global is not required.

Reply:
Noted. Thanks for pointing out.